frontend/src/pages/_middleware.ts

```diff
@@ -3,15 +3,20 @@ import { NextResponse } from "next/server";
 
 import { DEFAULT_LOCALE, VALID_LOCALES } from "~/locale/languages";
 
+const PUBLIC_FILE = /\.(.*)$/;
+
 export function middleware(req: NextRequest, _ev: NextFetchEvent) {
-  const { pathname } = req.nextUrl;
+  const shouldHandleLocale =
+    !PUBLIC_FILE.test(req.nextUrl.pathname) &&
+    !req.nextUrl.pathname.includes("/api/") &&
```

Comment:
I think we also have `/admin` (not sure if is set on next js as a redirect though)

Reply:
it is set in now.json but it seems it gets ignored now 🤔

Reply:
I wouldn't bother with it :D

Reply:
I fixed it, for some weird reason now.json (or vercel.json) were ignored, I moved it into next.config.js